### PR TITLE
Adds information on setting buffer size for eager message protocol

### PR DESCRIPTION
**Pull request by aturner-epcc**

### CODE REVIEW

File: docs/user-guide/tuning.md [hunk 1]
```diff
@@ -113,19 +113,25 @@ The advice for tuning the performance of `MPI_Send` is
...
 
 On ARCHER2, things are a little more complicated. Although the eager
-limit defaults to 16K bytes, messages up to 256K are sent
+limit defaults to 16KiB, messages up to 256KiB are sent
 asynchronously because they are actually sent as a number of smaller
 messages.
 
 To send even larger messages asynchronously, alter the value of
-`FI_OFI_RXM_SAR_LIMIT` in your SLURM script, e.g. to set to 512K:
+`FI_OFI_RXM_SAR_LIMIT` in your job submission script script, e.g. to set to 512KiB:
```
Comment (juanfrh) [inline docs/user-guide/tuning.md:121]: I think "script" is repeated twice.

File: docs/user-guide/tuning.md [hunk 2]
```diff
@@ -113,19 +113,25 @@ The advice for tuning the performance of `MPI_Send` is
...
 
     export FI_OFI_RXM_SAR_LIMIT=524288
 
+You can also change the size of the smaller messages that larger ones
+are broken up into by altering the value of `FI_OFI_RXM_BUFFER_SIZE`
```
Comment (juanfrh) [inline docs/user-guide/tuning.md:126]: Might be the following sentence more readable? _"You can also change the size of the smaller messages, **causing** that larger ones are broken up into, by altering the value of [...]"_

**Review by juanfrh — CHANGES_REQUESTED** [review]
I have two suggestions. The rest looks okay to me.

File: docs/user-guide/tuning.md (hunk 2 above)
Comment (kevinstratford) [reply docs/user-guide/tuning.md]: I've suggested a fix

**Review by kevinstratford — APPROVED** [review]

**Review by juanfrh — APPROVED** [review]
Thanks Kevin. LGTM.

**Pull request merged**